applications/GeoMechanicsApplication/tests/compressibility_tests/upw_updated_lagrange_fic/ProjectParameters.json

```diff
@@ -0,0 +1,152 @@
+        "min_iterations":                      6,
+        "max_iterations":                      15,
+        "number_cycles":                      1,
+        "reduction_factor":                   0.5,
+        "increase_factor":                    2.0,
+        "desired_iterations":                 4,
+        "max_radius_factor":                  10.0,
+        "min_radius_factor":                  0.1,
+        "calculate_reactions":                true,
+        "max_line_search_iterations":         5,
+        "first_alpha_value":                  0.5,
+        "second_alpha_value":                 1.0,
+        "min_alpha":                          0.1,
+        "max_alpha":                          2.0,
+        "line_search_tolerance":              0.5,
+        "rotation_dofs":                      false,
+        "linear_solver_settings":             {
+            "solver_type":   "LinearSolversApplication.sparse_lu",
+            "scaling":       true
+        },
+        "problem_domain_sub_model_part_list": ["Soil"],
+        "processes_sub_model_part_list":      ["BottomFixed","TopLoad","Soil"],
+        "body_domain_sub_model_part_list":    ["Soil"]
+    },
+    "output_processes": {
+        "gid_output": [{
+            "python_module": "gid_output_process",
+            "kratos_module": "KratosMultiphysics",
+            "process_name": "GiDOutputProcess",
+            "Parameters":    {
+                "model_part_name": "PorousDomain.porous_computational_model_part",
+                "output_name": "output",
+                "postprocess_parameters": {
+                    "result_file_configuration": {
+                        "gidpost_flags":       {
+                            "WriteDeformedMeshFlag": "WriteUndeformed",
+                            "WriteConditionsFlag":   "WriteElementsOnly",
+                            "GiDPostMode":           "GiD_PostAscii",
+                            "MultiFileFlag":         "SingleFile"
+                        },
+                        "file_label":          "step",
+                        "output_control_type": "step",
+                        "output_interval":    1,
+                        "body_output":         true,
+                        "node_output":         false,
+                        "skin_output":         false,
+                        "plane_output":        [],
+                        "nodal_results":       ["WATER_PRESSURE"],
+                        "gauss_point_results": []
+                    },
+                    "point_data_configuration":  []
+                }
+            }
+        }]
+    },
+    "processes": {
+        "constraints_process_list": [{
+        "python_module": "apply_vector_constraint_table_process",
+        "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+        "process_name":  "ApplyVectorConstraintTableProcess",
+        "Parameters":    {
+            "model_part_name": "PorousDomain.Soil",
```

Comment:
The whole domain is used to block horizontal displacements. That makes Poissons ratio meaningless. The intention is the side planes, by coincidence this is the whole domain as there is only 1 element. I would prefer to make a separate modelpart that only contains nodes and has an appropriate name for it.

Reply:
I agree it would be clearer. In this case that would mean I add a model part that also has all the nodes in it right?

Reply:
Done